submissions/Roka20012/pop-up-menu/index.html

```diff
@@ -34,8 +32,7 @@
             <div class="sprite"></div><span>No no..</span></a><a class="hide" href="">
```

Comment:
Good practice to fill empty `href` by attribute `#`. Example: `href="#"`

Reply:
Done)